docs/benchmark.md

```diff
@@ -12,86 +12,119 @@ It's important to look at the units than at the percentages. In general, RxDogTa
 
 Running the benchmark on a [Pixel 3](https://store.google.com/product/pixel_3_specs) yields the following results:
```

Comment:
Can't comment there but can you remove the note about percentages?

Reply:
Done in dde7232